_shared/hooks/post_gen_project.py

```diff
@@ -27,8 +27,8 @@ def remove_conditional_files():
     paths_to_remove.extend(["bin/make_devdata"])
     {% endif %}
 
-    {% if cookiecutter.get("services") != "yes" %}
-    paths_to_remove.extend(["docker-compose.yml", "requirements/dockercompose.in"])
+    {% if not (cookiecutter.get("postgres") == "yes" or include_exists("docker-compose/services.yml")) %}
+    paths_to_remove.extend(["docker-compose.yml"])
```

Comment:
This means that the cookiecutter will add the `docker-compose.yml` file to the project if _either_ `"postgres"` is `"yes"` in `cookiecutter.json` _or_ `.cookiecutter/includes/docker-compose/services.yml` exists.